Sources/Sentry/include/SentryFileManager.h

```diff
@@ -120,7 +120,8 @@ BOOL createDirectoryIfNotExists(NSString *path, NSError **error);
 /**
  * Path for a default directory Sentry can use in the app sandbox' caches directory.
  * @note This method must be statically accessible because it will be called during app launch,
- * before any instance of @c SentryFileManager exists, and so wouldn't be able to access this path
+ * before any instance of @c SentryFileManager exists, and so wouldn't be able to access this path.
+ * @note For unsandboxed macOS apps, the path has the form @c ~/Library/Caches/<app-bundle-id> .
```

Comment:
`m`: Isn't it prefixed with `io.sentry` after the fix?

```suggestion
 * @note For unsandboxed macOS apps, the path has the form @c ~/Library/Caches/io.sentry/<app-bundle-id> .
```

Reply:
This was wrong from my end.